src/main/java/net/schmizz/sshj/DefaultConfig.java

```diff
@@ -73,6 +73,7 @@
     public DefaultConfig() {
         setLoggerFactory(LoggerFactory.DEFAULT);
         setVersion(readVersionFromProperties());
+        SecurityUtils.setBouncycastleRuntimeRegistration(isBouncycastleRuntimeRegistrationEnabled());
```

Comment:
If you would add some tests, you'd see that you can never ever change the configuration flag, as it is already passed down in the constructor of the config if you do it like this.

Reply:
Thanks, you are right. Will address it asap

Reply:
any updates?